Dockerfile

```diff
@@ -10,13 +10,53 @@ COPY vendor/php-parser/composer.lock /usr/src/app/vendor/php-parser/
 
 COPY package.json /usr/src/app/
```

Comment:
The way the Dockerfile is currently written/architected, no. Bundle is part of the big build in order to get the package deletions to work. 

We can if we switch to a bunch of `RUN` lines and use that `--squash` command line option... but that needs experienced eyeballs on it.